app/controllers/api/v1/notifications_controller.rb

```diff
@@ -30,6 +30,11 @@ def clear
     render_empty
   end
 
+  def dismiss
+    Notification.where(account: current_account, id: params[:id]).delete_all
```

Comment:
Maybe `Notification.find_by!(account: current_account, id: params[:id]).destroy!` would be better here (throw 404 when bad ID, throw error if something wrong with deleting it)